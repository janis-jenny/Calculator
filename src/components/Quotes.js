import quoteStyle from '../style/quote.module.css';

const Quotes = () => (
  <div>
    <ul className={quoteStyle.list}>
      <li>“Technology is best when it brings people together.”</li>
      <li>“Keep going forward because success will come”</li>
      <li>“Engineering is the closest thing to magic that exists in the world.”</li>
    </ul>
  </div>
);

export default Quotes;
