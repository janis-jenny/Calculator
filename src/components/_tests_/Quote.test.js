import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quotes from '../Quotes';
import '@testing-library/jest-dom';

it('Should contain the head', () => {
  const { getByText } = render(<Quotes />);
  const head = getByText(/Technology/);
  expect(head).toBeInTheDocument();
});

it('Should contain the text', () => {
  const head = screen.queryByText('hello');
  expect(head).not.toBeInTheDocument();
});

describe('Quotes', () => {
  it('renders correctly', () => {
    const page = renderer.create(<Quotes />);
    expect(page).toMatchSnapshot();
  });
});
