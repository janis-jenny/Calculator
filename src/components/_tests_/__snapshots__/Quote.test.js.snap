// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders correctly 1`] = `
<div>
  <ul
    className="list"
  >
    <li>
      “Technology is best when it brings people together.”
    </li>
    <li>
      “Keep going forward because success will come”
    </li>
    <li>
      “Engineering is the closest thing to magic that exists in the world.”
    </li>
  </ul>
</div>
`;
